Update submodule.

Zangetsu38 opened the pull request:
Add branch info missing for submodule if needed.

Update Submodule on last tags stable version if it already used here, or last master if it already get it.
SPIRV-Cross->last master,
capstone->last tag 4.0.1,
elfio->last master,
gslang->last tag 7.10.2984,
imgui->last tag v1.67,
microprofil->last master,
sdl2-cmake-scripts->last master

VelocityRa commented:
Thanks, please change commit message to
```
external: Update some submodules
```

Zangetsu38 commented:
changed

VelocityRa commented:
I assume you built & tested it and it works fine?

Zangetsu38 commented:
Yes, tested some game and works normally

VelocityRa commented:
@scribam It's easier to determine which version/tag we're on, I suppose.

Zangetsu38 commented:
@scribam Is juste because master using this branch, but don't have information for that

scribam commented:
For me, adding `branch` seems only useful when you want to stick to a branch while using `git submodule update --remote`. I can understand the point to find the version/tag though.
I personally prefer to stay with the master branch to have all the fancy and cutting-edge changes if I want to :laughing:

Also, Capstone 4.0.1 and spdlog 1.3.0 are out if you want to be up to date Zangetsu38

Zangetsu38 commented:
@scribam I have changed nothing for branch added, it already using branch 1.23 and v3.0 on master, i just add info missing on gitmodule, that all ^^

Rebased and update again 3 submodule

Zangetsu38 commented:
return spdlog to 1.2.1 version like updated first time, 1.3.0 causing issue log broken.

VelocityRa commented on .gitmodules:
```diff
@@ -25,9 +26,11 @@
 [submodule "src/external/spdlog"]
 	path = src/external/spdlog
 	url = https://github.com/gabime/spdlog.git
+	branch = v1.x
```
remove this too

Zangetsu38 replied:
for what, i indicate just what branch is used that all ^^
